lib/events/events.go

```diff
@@ -238,6 +238,7 @@ type Event struct {
 type Subscription interface {
 	C() <-chan Event
 	Poll(timeout time.Duration) (Event, error)
+	Mask() EventType
```

Comment:
Maybe `Allows(EventType) bool` would be cleaner (just moving the `& ... != 0`)?

Reply:
`Allows` feels kind of weird in this context, and e.g. `SubscribedTo` would be even worse. The mask value is also used as a map key in the api, i.e. there could be a use-case for actually getting the value. I think I'll choose simple over clean here.